src/App.js

```diff
@@ -1,12 +1,11 @@
+import { SnackbarProvider } from "notistack";
 import UsersDetailsCard from "./components/UsersDetailsCard";
 import "./style.css";
 
 export default function App() {
   return (
-    <div className="App">
-      {/* <h1>Hello CodeSandbox</h1>
-      <h2>Start editing to see some magic happen!</h2> */}
+    <SnackbarProvider>
```

Comment:
Hi @UdayGoel2000 , you have to pass your component as children inside SnackbarProvider to start using its features.

read this for more reference - https://notistack.com/getting-started